README.md

```diff
@@ -354,6 +355,26 @@ _Mailing lists servers and mass mailing software - one message to many recipient
 - [SquirrelMail](http://squirrelmail.org) - Another browser-based IMAP client. ([Source Code](https://sourceforge.net/p/squirrelmail/code/HEAD/tree/)) `GPL-2.0` `PHP`
 - [WebMail Lite](http://www.afterlogic.org/webmail-lite) - Web-based IMAP Mail client. ([Source Code](https://github.com/afterlogic/webmail-lite)) `GPL-3.0` `PHP`
 
+#### Disposable email and forwarding services
+
+**[`^        back to top        ^`](#)**
+
+- [Ad Hoc Email Server (AHEM)](http://www.ahem.email) - An ad-hoc mail server. Create and use ad hoc disposable mailboxes for testing or any other purpose. http://www.ahem.email ([Demo](http://www.ahem.email/account/bi1vx86r), [Source Code](https://github.com/o4oren/Ad-Hoc-Email-Server)) `Angular/Nodejs` `MongoDB` `MIT`
+- [Alp-Mail](https://github.com/theju/alp-mail) - Alp-Mail (Alp in Hindi means short-lived) is your self-hosted disposable email service ala Mailinator. All mails are stored in a database under incoming or rejected. ([Source Code](https://github.com/theju/alp-mail)) `Nodejs` `Django` `MIT`
+- [disposable-mailbox](https://github.com/synox/disposable-mailbox) - self-hosted disposable email service (php) using catch-all and imap ([Demo](bhadoomail.com), [Source Code](https://github.com/synox/disposable-mailbox)) `PHP` `GPL-3.0`
+- [disposable](disposebox.com) - Web app for creating your own hassle free disposable mailbox service like Mailinator or TrashMail. ([Demo](disposebox.com), [Source Code](https://github.com/andris9/disposable)) `NodeJS` `MongoDB` `MIT`
+- [forsaken-mail](https://github.com/denghongcai/forsaken-mail) - a self-hosted disposable mail service ([Demo](http://disposable.dhc-app.com/), [Source Code](https://github.com/denghongcai/forsaken-mail)) `Docker` `NodeJS` `GPL-2.0`
```

Comment:
Don't duplicate links. Delete the link to the source code since the primary project link is in fact the same link to the source code.